lua/supermaven-nvim/completion_preview.lua

```diff
@@ -128,15 +128,15 @@ function CompletionPreview.on_accept_suggestion()
 
     vim.api.nvim_feedkeys(vim.api.nvim_replace_termcodes("<Space><Left><Del>", true, false, true), "n", false)
     vim.lsp.util.apply_text_edits({{ range = range, newText = completion_text }}, vim.api.nvim_get_current_buf(), "utf-16")
-    local adjust_cursor = "<End>"
-    vim.api.nvim_feedkeys(vim.api.nvim_replace_termcodes(adjust_cursor, true, false, true), "n", false)
+    local termcodes = string.rep("<Down>", u.line_count(completion_text))
```

Comment:
What does this change do?

Reply:
Goes down to the end of completion
instead of hitting <End> once and going to the end of current line
fixes #2

Reply:
Depending on the version of Neovim being used, it seems that `vim.lsip.util.apply_text_edits` already moves the cursor to the end of the line. I think feeding `<Down>` results in unintentionally pushing the cursor below the inserted completion for those versions